tests/storybook.ts

```diff
@@ -1,12 +1,22 @@
-import { runInRepo } from '../utils.ts'
+import { $, runInRepo } from '../utils.ts'
 import type { RunOptions } from '../types.d.ts'
+import { getCommand } from '@antfu/ni'
 
 export async function test(options: RunOptions) {
 	await runInRepo({
 		...options,
-		repo: 'storybookjs/builder-vite',
-		branch: 'main',
-		build: 'prepublish',
-		test: ['build-examples', 'test-ci'],
+		repo: 'storybookjs/sandboxes',
+		dir: 'next',
+		testSubdirectory: 'react-vite/default-ts/after-storybook',
+		branch: 'next',
+		build: ['build-storybook'],
+		beforeTest: async (agent) => {
```

Comment:
do we need to pass through agent here? a project should know it's preferred agent so just using `pnpm exeute...` here should work?